runtime/scripts.vim

```diff
@@ -306,7 +306,9 @@ else
     set ft=virata
 
     " Strace
-  elseif s:line1 =~# '^\(\[pid \d\+\] \)\=[0-9:.]* *execve(' || s:line1 =~# '^__libc_start_main'
+  elseif s:line1 =~# '^\(\[pid \d\+\] \)\=[0-9:.]* *execve('
+    \ || s:line1 =~# '^__libc_start_main'
+    \ || s:line1 =~# 'execve(.*)\s\+= \d\+'
```

Comment:
> Can you check, if that works for you?

Yes, this works for me. Please let me know if you need me to update the pull request or would prefer to add yourself, either this fix or the suggestion below (of simply checking for the existence of execve in the first line). Thanks for your time !